search_single_spectrum/test/test_foodomics.py

```diff
@@ -9,9 +9,9 @@ def test():
     input_filename = "data/foodomics/all_dataset_matchs.tsv"
```

Comment:
why is this change in this PR?

Reply:
I updated the branch to the latest master to ensure compatibility. This PR merges into the other library conversion workflow branch that is currently not up to date with the master.